spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -159,7 +159,6 @@ def __getattr__(self, attr):
                               is_cython=is_cython)
     window.setCentralWidget(console)
 
-    qtbot.addWidget(window)
```

Comment:
@ccordoba12 didn't you add this back to prevent segfaults?

Reply:
qtbot calls `deleteLater()` that then deletes the c++ object without deleting the python wrapper object

Reply:
Not on Windows. There each IPython console generated in a test remains visible, which at some hangs our tests.

It took me hours of trial an error to notice that, so please don't remove it.

Reply:
I'll try to move it to later. You are right that I only notice the segmentation faults in he macOS tests

Reply:
or alternatively this could be inside a test for window

Reply:
> or alternatively this could be inside a test for window

What do you mean by this?

Reply:
```
if platform == "win32":
    qtbot.addWidget(window)
```